keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/StudentT.java

```diff
@@ -0,0 +1,97 @@
+package io.improbable.keanu.distributions.continuous;
+
+import java.util.Random;
+
+import static java.lang.Math.*;
+import static org.apache.commons.math3.special.Gamma.gamma;
+import static org.apache.commons.math3.special.Gamma.digamma;
+
+public class StudentT {
+	/**
+	 * Computer Generation of Statistical Distributions
+	 * by Richard Saucier
+	 * ARL-TR-2168 March 2000
+	 * 5.1.23 page 36
+	 */
+	public static double sample(double v, Random random) {
+		assert( v > 0. );
```

Comment:
No need for assert. If it's necessary, throw an IllegalArgumentException with an appropriate message saying what the problem is.